lib/matplotlib/axis.py

```diff
@@ -120,7 +120,11 @@ def __init__(self, axes, loc, label=None,
         self._base_pad = pad
 
         if labelcolor is None:
-            labelcolor = mpl.rcParams[f"{name}.color"]
+            labelcolor = mpl.rcParams[f"{name}.labelcolor"]
+            if labelcolor is None:
```

Comment:
if you're using validate_color, this can't be none.  possibly you may want to use validate_color_or_inherit.

Reply:
Hi @anntzer thank you for your help. I just copied everything from the existing xtick.color into xtick.labelcolor (and xtick.color has the same `is None` check and the `validate_color`). Should that also change? I'm a bit confused.

Reply:
The argument passed in (`color`, `labelcolor`) can be None, but the rcparam cannot... right? (from a quick look)

Reply:
Thanks, I understand now! I changed it.

Reply:
I think this conditional can still be removed (as it is handled in the next block).

Reply:
You're absolutely right, I forgot that it can only be inherit or a color now. Thanks!